docs/src/operator/usage.md

```diff
@@ -32,6 +32,34 @@ miden-node bundled bootstrap \
   --accounts-directory .
 ```
 
+Alternatively, provide an explicit toml-formatted configuration which allows you to define
+multiple assets and faucets easily at chain startup.
+
+```sh
+miden-node bundled bootstrap \
+  --data-directory data \
+  --accounts-directory . \
+  --genesis-config-file genesis.toml
+```
+
+The genesis configuration file should contain at least one faucet, and optionally, wallet definitions with assets, for example:
+
+```toml
+timestamp = 1717344256
+version   = 1
+
+[[faucet]]
+decimals     = 12
+max_supply   = 10000
+storage_mode = "public"
+symbol       = "FUZZY"
+
+[[wallet]]
+assets       = [{ amount = 999, symbol = "FUZZY" }]
+storage_mode = "private"
+can_be_updated = false # default value
```

Comment:
Does this control the mutability of the accounts code?

@igamigo what do you call this in the client? So we can maybe align on terminology.

Reply:
I've seen mutable, but in practice that's updatable, or do we literally mutate the bytecode?